### DISCUSSION TITLE
How to add official support for s390x arch

### DISCUSSION THREAD

**Discussion by Repana-Chowdappa** [Kong/kong#12697] [Help]
Hi Team,
We have built the kong on s390x from https://github.com/Kong/kong using both bazel build tools and kong build tools.
Need to raise PR to merge the s390x code changes to the kong repo's to get kong supported on s390x.

**Comment by chronolaw** [comment]
@fffonion @Water-Melon , could you give some comments for this? thanks.

**Comment by hanshuebner** [comment]
@Repana-Chowdappa Please open a pull request so that we can review the changes to decide whether we can incorporate them.

Thank you!
Hans

**Reply by Repana-Chowdappa** [reply]
@hanshuebner I have open a pull request https://github.com/Kong/kong/pull/12750 for above request to add support for s390x. Please check and let us know if there is any changes required.
